parodos-model-api/src/main/java/com/redhat/parodos/workflow/context/WorkContextDelegate.java

```diff
@@ -15,6 +15,8 @@
  */
 package com.redhat.parodos.workflow.context;
 
+import static com.redhat.parodos.workflow.context.Visibility.*;
```

Comment:
can we expand the asterisk?

Reply:
sure, I'm surprised that happened with the IDE formatter.